pkg/packagekit/updates.less

```diff
@@ -117,26 +119,35 @@ tr.security.listing-ct-item {
 }
 
 
-/* Reformat table for narrow widths */
-@media screen and (max-width: 640px) {
+/* Hide changelog summaries in narrow widths */
+@media screen and (max-width: 80ch) {
+    .container-fluid {
+        overflow-x: hidden;
+    }
     table.listing-ct {
-        /* counteract .container-fluid */
-        margin: 0 -20px;
-        overflow: hidden;
-
-        /* Setting the width is bad for mobile, when undoing .container-fluid */
-        width: auto;
-
-        /* Don't shift size of cells, even when info is expanded */
-        table-layout: fixed;
-
         /* Hide changelog header (there's no class) & data */
         thead th:last-child,
         td.changelog {
             display: none;
         }
     }
 }
+/* Remove listing table margins in cramped space (mobile) */
+@media screen and (max-width: 640px) {
+    table.listing-ct {
+        /* counteract .container-fluid */
+        margin-left: -20px;
+        margin-right: -20px;
+
+        /* Mobile mode table should be full width */
+        width: 100vw;
+    }
+    tbody.open > tr.listing-ct-item,
+    tbody.open > tr.listing-ct-panel {
+        border-left: none;
+        border-right: none;
```

Comment:
Hide the activated row borders, as we're shaving off the sides of the table anyway.